aten/src/ATen/native/quantized/cpu/qlinear_dynamic.cpp

```diff
@@ -130,48 +102,47 @@ class QLinearDynamicInt8 final : public torch::OperatorKernel {
     auto output = at::empty(out_sizes, input.options().dtype(at::kFloat));
     auto buffer = at::empty_like(output, output.options().dtype(at::kInt));
 
-    if (pack_ptr.q_scheme == kPerTensorAffine) {
-      // Process the per tensor quantization.
+    int num_tasks = at::get_num_threads();
+    at::parallel_for(0, num_tasks, 1, [&](int64_t begin, int64_t end) {
```

Comment:
Is there a way to test the parallelization logic to see if there is a speedup?

Reply:
Will update with some performance numbers.

Reply:
Updated with the performance number in the summary.